packages/ui/src/working-groups/hooks/useApplications.ts

```diff
@@ -0,0 +1,32 @@
+import BN from 'bn.js'
+
+import { useMyMemberships } from '../../memberships/hooks/useMyMemberships'
+import { WorkingGroupOpening } from '../types'
+import { WorkingGroupApplication } from '../types/WorkingGroupApplication'
+
+export function useApplications(): WorkingGroupApplication[] {
```

Comment:
Hint: The usage pattern for all query node data will be:

```javascript
const { isLoading, foos } = useFoos()
```

So in the next iteration, you can change that as well.

For the layout setup phase, either is OK, but if you use the `isLoading + returnedData` pattern the fewer changes will be later.